src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -122,7 +122,8 @@
     <{{ element|default('div') }} class="custom-file">
         {%- set type = type|default('file') -%}
         {{- block('form_widget_simple') -}}
-        <label for="{{ form.vars.id }}" class="custom-file-label">
+        {%- set label_attr = label_attr|merge({class: (label_attr.class|default('') ~ ' custom-file-label')|trim}) %}
```

Comment:
Just a minor comment, if we add spaces before and after the curly brackets in the next line, we should do it here the same

Reply:
Changed (I hope you want to say that)